spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -121,7 +121,7 @@ def paginated_collection(*args)
       let(:pagination) { paginated_collection(collection, entry_name: "singular", entries_name: "plural") }
 
       it "should use :entry_name as the collection name" do
-        expect(pagination.find_by_class('pagination_information').first.content).to eq "Displaying <b>1</b> singular"
+        expect(pagination.find_by_class('pagination-information').first.content).to eq "Displaying <b>1</b> singular"
```

Comment:
Line is too long. [117/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.